content/protection.md

```diff
@@ -13,7 +13,7 @@ If your employer goes out of business you won’t lose your pension pot. This is
 
 ## If your provider can’t pay
 
-You can be compensated for up to 100% of the value of your pot if your pension provider can’t pay you and is authorised by the FCA.
+You can be compensated for up to 100% of the value of your pot if your pension provider can’t pay you and is authorised by the Financial Conduct Authority (FCA).
```

Comment:
Yep, good spot. I can't remember if there's markdown syntax for this. Will check first thing Monday.